script.js

```diff
@@ -0,0 +1,462 @@
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "enter":
+                    buttonElement.classList.add('enter');
+                    buttonElement.addEventListener('click', () => {
+                        this.properties.value += '\n';
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case " ":
+                    buttonElement.classList.add('space');
+                    buttonElement.addEventListener('click', () => {
+                        this.properties.value += ' ';
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "shift":
+                    buttonElement.classList.add('shift');
+                    buttonElement.addEventListener('click', () => {
+                        /*this.properties.value += ' ';*/
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "rshift":
+                    buttonElement.classList.add('shift');
+                    buttonElement.addEventListener('click', () => {
+                        /*this.properties.value += ' ';*/
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "ctrl":
+                    buttonElement.classList.add('ctrl');
+                    buttonElement.addEventListener('click', () => {
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "win":
+                        buttonElement.classList.add('win');
+                        buttonElement.addEventListener('click', () => {
+                            document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                        });
+                        break;
+
+                        case "alt":
+                        buttonElement.classList.add('alt');
+                        buttonElement.addEventListener('click', () => {
+                            document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                        });
+                        break;
+
+
+            
+            
+            default:
+            buttonElement.addEventListener('click', () => {
+                if (this.properties.ru == 'false') {
```

Comment:
I think it was better to find another solution to distinguish languages because you have boolean values but you use them as strings. you can just use 'ru' and 'en'. Using true/false make it difficult to add one more language. Due to the only two possible variants

Reply:
Yes, it sounds much better, even because i use 'true' and 'false' as a strings in the task.